runtime/syntax/help.vim

```diff
@@ -85,7 +85,6 @@ syn match helpSpecial		"CTRL-PageDown"
 syn match helpSpecial		"CTRL-Insert"
 syn match helpSpecial		"CTRL-Del"
 syn match helpSpecial		"CTRL-{char}"
-syn region helpNotVi		start="{Vi[: ]" start="{not" start="{only" end="}" contains=helpLeadBlank,helpHyperTextJump
 syn match helpLeadBlank		"^\s\+" contained
```

Comment:
As far as I can see, `helpLeadBlank` is now no longer needed and can be deleted. True?